### PR TITLE
Alusine - JavaScript2 week2

### PR DESCRIPTION
**Pull request by alquinex**

### CODE REVIEW

File: Week2/homework/maartjes-work.js [hunk 1]
```diff
@@ -46,16 +46,24 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
+  const durationInHours = tasks.map(task => task.duration / 60);
+  const filterHours = durationInHours.filter(hour => hour >= 2);
+  for (let i = 0; i < filterHours.length; i++) {
```
Comment (hasan-sh) [inline Week2/homework/maartjes-work.js]: Do you really need to use this for loop? Would perhaps chaining the `map` method work instead?

File: Week2/homework/map-filter.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
-  console.log(numbers);
+  const newNumbers = numbers.filter(number => number % 2 !== 0).map(number => number * 2);
```
Comment (hasan-sh) [inline Week2/homework/map-filter.js]: Very nice how you chained the two methods :+1:

File: Week2/homework/maartjes-work.js (hunk 1 above)
Comment (alquinex) [reply Week2/homework/maartjes-work.js]: Hi Hasan,
Thanks for taking the time to go through my homework. Changes have been made.

**Comment by NoerGitKat** [comment]
Thanks for the feedback @hasan-sh ! And great job @alquinex , you're back in the game!

**Pull request closed without merging**